cmd/createcluster.go

```diff
@@ -88,7 +88,7 @@ func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.IntVarP(&config.Threshold, "threshold", "t", 3, "The threshold required for signature reconstruction. Minimum is n-(ceil(n/3)-1).")
 	flags.StringVar(&config.FeeRecipient, "fee-recipient-address", "", "Optional Ethereum address of the fee recipient")
 	flags.StringVar(&config.WithdrawalAddr, "withdrawal-address", defaultWithdrawalAddr, "Ethereum address to receive the returned stake and accrued rewards.")
-	flags.StringVar(&config.Network, "network", defaultNetwork, "Ethereum network to create validators for. Options: mainnet, prater, kintsugi, kiln, gnosis.")
+	flags.StringVar(&config.Network, "network", defaultNetwork, "Ethereum network to create validators for. Options: mainnet, prater, kintsugi, kiln, ropsten, gnosis.")
```

Comment:
discussion: we can remove `kintsugi` 
<img width="1047" alt="Screenshot_2022-07-28_at_2 49 30_PM" src="https://user-images.githubusercontent.com/43061995/183354963-20222d65-1026-4c89-98dc-ed8563eac988.png">

cc: @corverroos